CRM/Utils/Array.php

```diff
@@ -38,6 +38,10 @@ public static function value($key, $list, $default = NULL) {
     if (is_array($list)) {
       return array_key_exists($key, $list) ? $list[$key] : $default;
     }
+    if ($list instanceof ArrayAccess) {
+      return $list[$key];
```

Comment:
Shouldn't this be
```suggestion
      return $list[$key] ?? $default;
```

Reply:
I guess -the issue is above array handling treats a set but NULL key as higher than the default. This won't but it's probably better to do as you say anyway